README.md

```diff
@@ -161,7 +160,7 @@ In this section, we'll use the [Azure CLI](https://docs.microsoft.com/cli/azure/
    az postgres db create --resource-group $rgName --name $dbName --server-name $serverName
 
    # Allow Azure resources through the firewall
-   az postgres server firewall-rule create --resource-group $rgName --server-name $serverName --name AllowAllAzureIps --start-ip-range 0.0.0.0 --end-ip-range 0.0.0.0
+   az postgres server firewall-rule create --resource-group $rgName --server-name $serverName --name AllowAllAzureIps --start-ip-address 0.0.0.0 --end-ip-address 0.0.0.0
```

Comment:
This appears to be `address` instead of `range`.

<img width="815" alt="Pasted Graphic 2" src="https://user-images.githubusercontent.com/737821/138193169-678f2a58-6046-41ad-a666-cbe4ae4ec1f0.png">

Reply:
I best check out docs and make sure they aren't wrong too